docs/source/features/playground.md

```diff
@@ -0,0 +1,45 @@
+---
+title: GraphQL Playground
+description: Visually exploring a Apollo Server
+---
+
+[GraphQL Playground](https://github.com/prismagraphql/graphql-playground) is a graphical interactive in-browser GraphQL IDE, created by [Prisma](https://www.prisma.io/), based on [GraphiQL](https://github.com/graphql/graphiql). In development, Apollo Server collocates a GraphQL Playground instance with the GraphQL path. When a browser sends a request to Apollo Server, it receives the GraphQL Playground gui. When `NODE_ENV` is set to production, introspection and Playground are disabled as a production best practice.
+
+<div align="center">
+![GraphQL Playground](../images/playground.png)
+</div>
+
+## Enabling Playground in Production
+
+To enable Playground in production, an integration package must be installed to provide more control over the middlewares used. The following example uses the express integration:
+
+```bash
+npm uninstall --save apollo-server
```

Comment:
I'd recommend that we do not show the uninstallation of apollo-server. It's really not needed here.